packages/core/upload/server/register.js

```diff
@@ -73,4 +74,9 @@ const baseProvider = {
   extend(obj) {
     Object.assign(this, obj);
   },
+  checkFileSize(file, { sizeLimit }) {
+    if (sizeLimit && kbytesToBytes(file.size) > sizeLimit) {
+      throw new PayloadTooLargeError();
```

Comment:
About the custom message, we could just do ```throw new PayloadTooLargeError(`${file.name} is too big`);``` here, wdyt?

If we wanted, we could go one step further and also say what the file limit is.

PS: it might not be `file.name`, it was just to outline the idea.

Reply:
Ok!! What do you think of something like this. Using a size limit of 4MB, you get this error in the response:
<img width="535" alt="image" src="https://user-images.githubusercontent.com/20578351/191963006-6c820471-b8a0-46b9-a40b-75436aee1aea.png">

Reply:
I like it!